ui/src/containers/AppVersionTableContainer.js

```diff
@@ -55,31 +52,29 @@ export class AppTableContainer extends React.Component {
   };
 
   render () {
-    if (this.props.isAppsRequestFailed) {
+    if (!this.props.appVersions) {
       return (
-        <div className="no-apps">
-          <p>Unable to fetch any apps</p>
+        <div className="no-versions">
+          <p>This app has no versions</p>
         </div>
       );
     }
-    return this.getTable();
+
+    return this.getTable(this.props.appVersions);
   }
 }
 
-AppTableContainer.propTypes = {
-  app: PropTypes.array.isRequired,
+AppVersionTableContainer.propTypes = {
   sortBy: PropTypes.object.isRequired,
-  columns: PropTypes.array.isRequired,
-  isAppsRequestFailed: PropTypes.bool.isRequired
+  columns: PropTypes.array.isRequired
 };
 
 function mapStateToProps (state) {
   return {
-    app: state.appDetailRows,
-    sortBy: state.appDetailsSortDirection,
-    columns: state.appDetailColumns,
-    isAppsRequestFailed: state.isAppsRequestFailed
+    sortBy: state.appVersionsSortDirection,
+    columns: state.appVersionsColumns,
+    appVersions: state.app.deployedVersions.rows
```

Comment:
Should we include also appVersions under `AppVersionTableContainer.propTypes` above?

Reply:
Would that fail when appVersions is an empty array or is that valid?

Reply:
The propType check shouldn't - my understanding is it just checks for the presence of that prop and triggers a warning if it's not

Reply:
Updated.